near/omni-types/src/lib.rs

```diff
@@ -556,7 +550,8 @@ impl FastTransfer {
     }
 }
 
-#[derive(BorshDeserialize, BorshSerialize, Debug, Clone)]
+#[near(serializers=[borsh, json])]
```

Comment:
Is json needed here?

Reply:
For now no, but if we are going to add an getter then yes